out/.roku-deploy-staging/components/tasks/http.brs

```diff
@@ -0,0 +1,75 @@
+sub init()
```

Comment:
remove the out folder from the git repo
add this folder to .gitignore file

Reply:
Done